internals/controllers/migrationhierarchy_controller.go

```diff
@@ -138,3 +155,14 @@ func (r *MigrationHierarchyReconciler) SetupWithManager(mgr ctrl.Manager) error
 		For(&danav1.MigrationHierarchy{}).
 		Complete(r)
 }
+
+//the function get two paramaters: snsName and snsNamespace
+//then add the sns to the sns event channel for trigger new resync for the sns 
```

Comment:
The comment above a function should start with the name of the function. Like this:

"`addSnsToSnsEvent` takes two parameters..."